source/features/merge-status.tsx

```diff
@@ -0,0 +1,64 @@
+import './merge-status.css';
+import select from 'select-dom';
+import React from 'dom-chef';
+import * as api from '../libs/api';
+import {getOwnerAndRepo} from '../libs/utils';
+import features from '../libs/features';
+
+const CONFLICTING = 'CONFLICTING';
+
+function buildQuery(
+	ownerName: string,
+	repoName: string,
+	prs: string[],
+): string {
+	return `
+		query {
+			repository(owner: "${ownerName}", name: "${repoName}") {
+				${prs.map(
+		(pr: string) => `
+					${pr}: pullRequest(number: ${pr.replace('issue_', '')}) {
+						mergeable
+					}
+				`,
```

Comment:
This feature can also run on the global “Pull requests” page you find in the black header bar, so it should account for multiple repos as well. You can get the user/repo from the PR link itself

Reply:
I didn't know the `global pull requests` page existed TBH. I'll implement it also for that page.

Reply:
It would be this: https://github.com/pulls

I think the `features.isPRList` detection should always include it but it doesn't. 


https://github.com/sindresorhus/refined-github/blob/c00058d1afe2be3d0921fb54b31161c8b58112f0/source/libs/page-detect.ts#L63

Should be:

```js
export const isPRList = (): boolean => location.pathname === '/pulls' || getRepoPath() === 'pulls';
```

Reply:
Should I change it? It might break other features that rely on this helper function.

Reply:
I looked at the [other features](https://github.com/sindresorhus/refined-github/search?q=isPRList&unscoped_q=isPRList) and yes you can change it.

One feature should also work on the new page and one feature will show an error in the console. Someone will fix it